plugins/topology/package.json

```diff
@@ -41,7 +41,7 @@
     "@patternfly/react-icons": "^4.93.6",
     "@patternfly/react-styles": "^4.92.6",
     "@patternfly/react-tokens": "4.94.6",
-    "@patternfly/react-topology": "4.93.6",
+    "@patternfly/react-topology": "5.0.0",
```

Comment:
This is a breaking change. We are handling this via https://github.com/janus-idp/backstage-plugins/pull/715